src/analysis.ts

```diff
@@ -47,6 +48,11 @@ export function validateResources(template: CloudFormationTemplate): ResourceTyp
           ResourceType: resourceType,
           FollowsBestPractice: false,
         };
+      case S3Bucket.resourceType:
+        return {
+          ResourceType: resourceType,
+          FollowsBestPractice: !new Set(Object.values(S3Bucket.validate(template))).has(false),
```

Comment:
This line feels a bit confusing. Perhaps just `Object.values(...).includes(false)`?

Reply:
Agreed! Will refactor into a utility function as this line is repeated a few times now.